rust/Cargo.toml

```diff
@@ -10,3 +10,8 @@ crate-type = ["cdylib", "staticlib"] # Android needs cdylib, whereas iOS staticl
 [dependencies]
 anyhow = { version = "1", features = ["backtrace"] }
 flutter_rust_bridge = "1.49.0"
+bip39 = "1.0.1"
+rand = "^0.6.0"
+hkdf = "0.12"
+sha2 = "0.10"
+bdk = "0.23.0"
```

Comment:
note to self: I forgot to enable dprint for toml files 🤦‍♂️ should be fixed in a sec :)

Reply:
this is fixed, you can rebase and run `dprint fmt` on it :)